vrplib/parse/parse_vrplib.py

```diff
@@ -33,17 +32,17 @@ def parse_vrplib(text: str, compute_edge_weights: bool = True) -> Instance:
     dict
         The instance data.
     """
-    instance = {}
+    instance = Instance()
 
     specs, sections = group_specifications_and_sections(text2lines(text))
 
     for spec in specs:
         key, value = parse_specification(spec)
-        instance[key] = value
+        instance[key] = value  # type: ignore
 
     for section in sections:
         section, data = parse_section(section, instance)
-        instance[section] = data
+        instance[section] = data  # type: ignore
```

Comment:
I'm not sure if it even makes sense to use a TypedDict if I ignore this line. TypedDict requires string literals.

Reply:
It's OK. Even though this line is no longer checked, the returned instance is assumed to be of type `Instance`, which used by mypy to do static type hinting.

Reply:
But if we do this, we also need to validate that the instance indeed follows the types as specified in `Instance`.

Reply:
Maybe we can use `pydantic` for this?

Reply:
Make a new `validate` module that validates the instance and solution outputs according to VRPLIB.